.changeset/strange-tomatoes-float.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/cli': minor
+---
+
+Use TS uploader for shopify theme share command
```

Comment:
(optional) Personally, I'd try to keep the changelogs a bit user-centric, so I'd update with something like this:

> Use the new implementation of `shopify theme share` that no longer depends on Ruby to work. Additionally, it fixes the command so it no longer ignores `.shopifyignore`.

As a user, it's not so important that the uploader is in TS (TypeScript), it's that they no longer need Ruby on their machine. Also, it's important that `.shopifyignore` now works as expected (so I'd focus a bit more on those aspects).

Reply:
I like it. I'll adjust the wording. Thank you for the feedback!